README.md

```diff
@@ -296,10 +296,17 @@ var helper = algoliasearchHelper(client, indexName, {
 });
 ```
 
-##### Add a numeric refinement
+##### Add numeric refinements
 
 ```js
 helper.addNumericRefinement('numericFacet', '=', '3').search();
```

Comment:
We need to replace `numericFacet` by `numericAttribute`, it's kind of misleading otherwise.